libs/sdk-core/src/models.rs

```diff
@@ -640,7 +640,7 @@ pub struct Payment {
 pub struct PaymentExternalInfo {
     pub lnurl_pay_success_action: Option<SuccessActionProcessed>,
     pub lnurl_metadata: Option<String>,
-    pub ln_address: Option<String>,
+    pub ln_address_or_domain: Option<String>,
```

Comment:
@andrei-21  In LnPaymentDetails we added another field for domain, why not adding here as well instead of reusing the same field for both values?